cmake/thirdparty/SetupThirdParty.cmake

```diff
@@ -115,3 +115,21 @@ if(CMAKE_GENERATOR STREQUAL "Unix Makefiles" OR CMAKE_GENERATOR STREQUAL "Ninja"
     blt_find_executable(NAME        ClangTidy
                         EXECUTABLES clang-tidy)
 endif()
+
+#------------------------------------
+# Code coverage
+#------------------------------------
+if (ENABLE_COVERAGE)
+    # find_executable requires that the executables be enabled
+    set(ENABLE_GCOV ON)
```

Comment:
Could you please clarify how this line interacts with the corresponding cache variable `ENABLE_GCOV` if it is present?

My understanding is that it locally overrides that value (e.g. for the `blt_find_executable` below), but will not modify the CACHE variable's value, e.g. for use elsewhere in the code.

Reply:
Right now these are only needed for compatibility with `blt_find_executable` as discussed above (and not used anywhere else), but I think it's reasonable to set/use the cached variable instead - if this coverage functionality is extended further or if the user just wants to disable a particular tool it might be unwise to unconditionally set the "local" variable.